pkg/cmd/cli/cmd/deploy.go

```diff
@@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	config.Status.LatestVersion++
 	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
 	if err == nil {
-		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
+		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
+		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
```

Comment:
We could probably just drop it.

On Tue, Apr 5, 2016 at 1:46 PM, Michail Kargakis <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/deploy.go
> <https://github.com/openshift/origin/pull/8360#discussion_r58583055>:
>
> > @@ -227,7 +228,9 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
> >  	config.Status.LatestVersion++
> >  	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
> >  	if err == nil {
> > -		fmt.Fprintf(out, "Started deployment #%d\n", config.Status.LatestVersion)
> > +		fmt.Fprintf(out, "--> Started deployment #%d\n", config.Status.LatestVersion)
> > +		fmt.Fprintf(out, "    Use '%s logs -f dc/%s' to track its progress.\n", o.baseCommandName, config.Name)
> > +		fmt.Fprintf(out, "    Use '%s scale --replicas=N dc/%s' to scale it accordingly.\n", o.baseCommandName, config.Name)
>
> I don't know that this is the right place to suggest this. Why would I
> scale while a deployment is running?
>
> Right, I added it mostly to replace "or even scale multiple deployments up
> at the same time." which I removed from the help text.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8360/files/6fda297aedd34d67eb786c818ff0a59e7dcc0c7b#r58583055>
>